x-pack/platform/plugins/shared/onechat/server/plugin.ts

```diff
@@ -73,6 +75,11 @@ export class OnechatPlugin
 
     registerUISettings({ uiSettings: coreSetup.uiSettings });
 
+    if (setupDeps.workflowsExtensions) {
+      registerStepDefinitions(setupDeps.workflowsExtensions);
+      this.logger.info('Onechat workflow step definitions registered');
```

Comment:
NIT: is the log line really useful? (personal opinion, it's just that I hate all the fleet + security solution startup messages)